server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/platform/PlatformAware.java

```diff
@@ -13,10 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kaaproject.kaa.server.operations.service.akka.messages.io;
+package org.kaaproject.kaa.server.transport.platform;
 
+/**
+ * Marker interface indicating that a class has a basic knowledge about related
```

Comment:
A marker interface indicating that a class has basic knowledge about the related platform layer that is used to encode/decode messages.